docs/governance/governance.md

```diff
@@ -0,0 +1,241 @@
+
+EESSI GitHub repository maintainers are responsible for reviewing and merging PRs.
+
+### 2.3 EESSI Infrastructure maintainers
+EESSI Infrastructure maintainers are those individuals that maintain the Central Server for the EESSI CernVM-FS repository (the CernVM-FS Stratum 0), the _public_ mirror servers (CernVM-FS Stratum 1's), and EESSI build infrastructure (e.g. hosting the EESSI build bot).
+
+Infrastructure maintainers are responsible for monitoring and maintaining their respective infrastructure, and provide access to those who need it according to the Roles and Responsibilities described here. Note that maintainers of build infrastructure have the right to limit access to a _subset_ of the Deployers and Builders as described in 2.4 and 2.5. Furthermore, they are _not_ allowed to give access to others that are not part of the EESSI Infrastructure maintainers, Deployers or Builders teams.
+
+### 2.4 Builders { #builders }
+Adding software to EESSI requires three steps (see [the contribution workflow](../adding_software/overview.md) documentation):
+
+1. The software needs to be built (compiled);
+2. A tarball of the software installation needs to be uploaded to a central location (e.g. S3 bucket) where the Central Server for the EESSI CernVM-FS repository can pick it up;
+3. The Central Server for the EESSI CernVM-FS repository needs to ingest the tarball.
+
+Builders are those individuals that have permissions to build software through one or more of the EESSI build bots, i.e. step 1 in this process.
+
+Builders are responsible for reviewing contributions. They should, for example, check that contributions adhere to the contribution guidelines, that a contribution will not trigger execution of any unintended or unwanted code, etc. They are also responsible for triggering builds for contributors who want to add software to the EESSI software stack.
+
+### 2.5 Deployers
+Deployers are those individuals that have permissions to deploy software through one or more of the EESSI build bots to a central location (e.g. S3 bucket), i.e. they are essentially responsible for step 2 as it is described in the [Builders](#builders) Section.
+
+Deployers should check the file list of the produced tarballs (as it is reported by the build bot) for unexpected items. They should also check if all officially supported hardware targets have been built for.
+
+### 2.6 Ingesters
+Ingesters are those individuals that have permissions to ingest a tarball into the EESSI CernVM-FS repository, i.e. they are essentially responsible for step 3 as it is described in the [Builders](#builders) Section.
+
+Ingesters have merge permissions on the (private) `EESSI/staging` GitHub repository.
+
+Ingesters should check the contents of the tarball as it is reported in the pull requests in the `EESSI/staging` repository. This essentially provides a second check (next to that done by Deployers) on the contents of the tarball.
```

Comment:
Recently, we started using `staging_bundles`. Maybe `EESSI/staging*` is more flexible?

Reply:
```suggestion
Ingesters should check the contents of the tarball as it is reported in the pull requests in the staging repository. This essentially provides a second check (next to that done by Deployers) on the contents of the tarball.
```